src/sharenodekeys.cpp

```diff
@@ -49,11 +49,11 @@ void ShareNodeKeys::add(Node* n, Node* sn, int specific)
         sn = n;
     }
 
-    add((NodeCore*)n, sn, specific);
+    add(n->nodekey(), n->nodehandle, sn, specific, NULL, 0);
```

Comment:
[no showstopper]: `NULL, 0` are default. A plus, we could use some comments on the prototypes there

Reply:
thanks- updated both aspects.  cheers